src/Clap-Core.package/ClapCommand.class/README.md

```diff
@@ -0,0 +1,18 @@
+I am a class which represents all elements(flags, positionals...) of a command, i.e. all elements this command can accept.
+
+To create a new instance of this class, you must write this:
+	ClapCommand withName: <commandName>
+	
+And after, if you want to add a flag:
+	addFlag: <flagName>
```

Comment:
I should check but from memory this is wrong, the parameter should be a proper `Flag` object not a string. Compare with the examples in the tests